tests/test_core.py

```diff
@@ -849,15 +849,15 @@ def test_mallocstruct(self):
   @no_lsan('LSan does not support custom memory allocators')
   @parameterized({
     'normal': [],
-    'debug': ['-DEMMALLOC_DEBUG'],
-    'debug_log': ['-DEMMALLOC_DEBUG', '-DEMMALLOC_DEBUG_LOG', '-DRANDOM_ITERS=130'],
+    'memvalidate': ['-DEMMALLOC_MEMVALIDATE'],
+    'memvalidate_verbose': ['-DEMMALLOC_MEMVALIDATE', '-DEMMALLOC_VERBOSE', '-DRANDOM_ITERS=130'],
   })
   def test_emmalloc(self, *args):
     # in newer clang+llvm, the internal calls to malloc in emmalloc may be optimized under
     # the assumption that they are external, so like in system_libs.py where we build
     # malloc, we need to disable builtin here too
     self.set_setting('MALLOC', 'none')
-    self.emcc_args += ['-fno-builtin'] + list(args)
+    self.emcc_args += ['-fno-builtin', '-g'] + list(args)
```

Comment:
It helps diagnosing the error messages that are reported when the test fails, instead of getting the wasm-function[...]s. In general whenever we have tests that are written in a form "need to edit the test to debug any failures in it", I think it is of a bit bad form? Although can certainly drop if there is more sense in testing non `-g` variant in general.

Reply:
Dropped `-g`

Reply:
I tend to agree.  I end up added `-g` to tests all the time.  But we should probably solve the in a general way.    Its certainly annoying when you see test failures in the CI and they don't have useful backtraces.

We should measure the cost of add `-g` by default and consider doing it.

Reply:
`-g` may be too much (full DWARF), if all we want are stack traces. `-g2` of `--profiling-funcs` is enough for traces.

But II'm not sure we want it by default in all tests, as that isn't the default way users run code?